flask_mongoengine/wtf/models.py

```diff
@@ -4,12 +4,12 @@
 class ModelForm(FlaskForm):
     """A WTForms mongoengine model form"""
 
-    def __init__(self, formdata=None, obj=None, **kwargs):
-        self.instance = (kwargs.pop('instance', None) or obj)
+    def __init__(self, formdata=None, **kwargs):
+        self.instance = (kwargs.pop('instance', None) or kwargs.get('obj'))
```

Comment:
...thinking about this some more, this should probably be ``kwargs.get('obj', None)`` to avoid a ``KeyError`` being raise seeing as this is an optional parameter.

Do I need to submit a new PR or will somebody with commit access just make the change?